code/modules/mod/modules/modules_visor.dm

```diff
@@ -0,0 +1,93 @@
+//Visor modules for MODsuits
+
+///Base Visor - Adds a specific HUD and traits to you.
+/obj/item/mod/module/visor
+	name = "MOD visor module"
+	desc = "A heads-up display installed into the visor of the suit. They say these also let you see behind you."
+	module_type = MODULE_TOGGLE
+	complexity = 1
+	active_power_cost = DEFAULT_CHARGE_DRAIN * 0.3
+	incompatible_modules = list(/obj/item/mod/module/visor)
+	cooldown_time = 0.5 SECONDS
+	/// The HUD type given by the visor.
+	var/hud_type
+	/// The trait given by the visor.
+	var/visor_trait = list()
+
+/obj/item/mod/module/visor/on_activation()
+	. = ..()
+	if(!.)
+		return
+	if(hud_type)
+		var/datum/atom_hud/hud = GLOB.huds[hud_type]
+		hud.add_hud_to(mod.wearer)
+	if(length(visor_trait))
+		ADD_TRAIT(mod.wearer, visor_trait, "mod_trait")
```

Comment:
Can you create a `MODSUIT_TRAIT` define for modsuit traits? And use that in place of all `"mod_trait"`.

Reply:
done